### PR TITLE
[release calendar] Add React and Webpack for client side

### PR DESCRIPTION
**Pull request by estherkim**
To debug, run `npm run client` which bundles the files in `src/client/` and serves the website on `localhost:9000` using webpack-dev-server.

Closes https://github.com/ampproject/amp-github-apps/issues/723

### CODE REVIEW

File: release-calendar/package.json [hunk 1]
```diff
@@ -10,30 +10,44 @@
...
+    "lint": "eslint --quiet --fix src/* test/* --ext .ts,.tsx ",
     "start": "npm run build && node -r dotenv/config ./dist/app.js",
-    "test": "jest --verbose=false"
+    "test": "jest --verbose=false",
+    "client": "npm run client-build && npm run client-serve",
+    "client-build": "webpack --config ./webpack.client.config.js",
+    "client-serve": "webpack-dev-server --mode development --open --hot --config ./webpack.client.config.js",
   },
   "dependencies": {
-    "dotenv": "8.2.0",
```
Comment (ajwhatson) [inline release-calendar/package.json]: why did you remove this?

Comment (estherkim) [reply release-calendar/package.json]: I moved it below under `devDependencies`

Comment (ajwhatson) [reply release-calendar/package.json]: oh, duh, haha

Comment (danielrozenberg) [reply release-calendar/package.json]: @estherkim _is it_ a dev dependencies? We use `dotenv` as a code dependency in other projects so I'm just wondering if this is not the case here either... what is it used for?

Comment (estherkim) [reply release-calendar/package.json]: It's to replace `process.env` variables. One way is to run it at app start like `require('dotenv').config()` so maaaaybe you need it in dependencies?

But I used the CLI way and put it in a node script like `node -r dotenv/config ./dist/app.js`. So as long as the variables get replaced before `dist/` is deployed then I think it's okay. I thiiiink

**Comment by danielrozenberg** [comment]
We'll find out :)
